pkg/make/builder.go

```diff
@@ -287,29 +287,38 @@ func (b *Builder) GenerateTUF(ctx context.Context) error {
 	}
 
 	notaryConfigDir := filepath.Join(fs.Gopath(), "src/github.com/kolide/launcher/tools/notary/config")
-	notaryConfigFile, err := os.Open(filepath.Join(notaryConfigDir, "config.json"))
-	if err != nil {
-		return errors.Wrap(err, "opening notary config file")
-	}
-	defer notaryConfigFile.Close()
-	var conf struct {
-		RemoteServer struct {
-			URL string `json:"url"`
-		} `json:"remote_server"`
+	notaryURL := os.Getenv("NOTARY_URL")
```

Comment:
Just throwing my 2c in, but IMO os.Getenv should be restricted to func main just like flags. Side effects in the middle of code are difficult to test and debug.